services/motion/builtin/replanner.go

```diff
@@ -14,12 +17,12 @@ type replanResponse struct {
 // replanner bundles everything needed to execute a function at a given interval and return.
 type replanner struct {
 	period       time.Duration
-	fnToPoll     func(ctx context.Context) replanResponse
+	fnToPoll     func(context.Context, [][]referenceframe.Input, int) (bool, error)
```

Comment:
Given we use the type `func(context.Context, [][]referenceframe.Input, int) (bool, error)` in multiple places could we please give it a type alias?

Reply:
I had one but its not really that useful as it only can be used in one place, which is in the line above.  In general function type aliases are not as useful i go as I want them to be

Reply:
I could add it back I suppoes